cmd/node/main.go

```diff
@@ -39,6 +41,9 @@ import (
 const (
 	defaultLogPath     = "logs"
 	defaultStatsPath   = "stats"
+	defaultDBPath      = "db"
+	defaultEpochString = "Epoch_"
```

Comment:
_ can be left out and changed in the below Sprintf usage. See below comment. ("%s_%d")

Reply:
changed